charts/datadog/templates/cluster-agent-deployment.yaml

```diff
@@ -216,6 +216,12 @@ spec:
           - name: DD_COMPLIANCE_CONFIG_CHECK_INTERVAL
             value: {{ .Values.datadog.securityAgent.compliance.checkInterval | quote }}
           {{- end }}
+          {{- if .Values.datadog.prometheusScrape.enabled }}
```

Comment:
Hmm, I think that this `if` block should be put after the last `end` block.

At this moment, when I enable `prometheusScrape` nothing happens. I see that this `if` is in another block (`{{- if eq  (include "should-enable-security-agent" .) "true" }}`).

Reply:
I can create PR for this.

Reply:
Indeed, I fixed it part of this PR https://github.com/DataDog/helm-charts/pull/201 actually which will be merged when Agent 7.27 is released.
But this fix itself can be fixed and merged immediately without waiting for the Agent release, so feel free to submit a fix 😄  Thanks!

Reply:
Hey 🖐

Here is the fix: https://github.com/DataDog/helm-charts/pull/202